### PR TITLE
[JUJU-4351] Remove unused DefaultSeries/Base from modelmanager facades returns

### PR DESCRIPTION
**Pull request by jack-w-shaw**
Remove unused DefaultSeries/Base from modelmanager facades returns

These properties are unused anywhere in the client. Dropping them
simplifies things, and makes steps towards removing series

Drop these from the returns of ModelInfo, ListModelSummaries and CreateModel
facades

## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- ~[ ] Comments saying why design decisions were made~
- [x] Go unit tests, with comments saying what you're testing
- ~[ ] [Integration tests](https://github.com/juju/juju/tree/main/tests), with comments saying what you're testing~
- ~[ ] [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages~

## QA steps

```sh
juju bootstrap lxd lxd
juju add-mode m
juju list-models
juju show-model m
```

### CODE REVIEW

File: api/base/types.go [hunk 1]
```diff
@@ -125,7 +125,6 @@ type UserModelSummary struct {
 	ControllerUUID     string
 	IsController       bool
 	ProviderType       string
-	DefaultSeries      string
```
Comment (SimonRichardson) [inline api/base/types.go:128]: Should we return `DefaultBase` instead?

Comment (jack-w-shaw) [reply api/base/types.go:128]: No need really imo. We won't use it anywhere

**Review by SimonRichardson — APPROVED** [review]
Looking into this more, we see that we use the model config to find the base/series so we can remove this.

File: apiserver/facades/client/modelmanager/modelmanager.go [hunk 2]
```diff
@@ -939,19 +937,6 @@ func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag, withSecrets bool) (pa
...
-		defaultBase := config.PreferredBase(cfg)
-		info.DefaultBase = defaultBase.String()
-		if defaultSeries, err := series.GetSeriesFromBase(defaultBase); err == nil {
-			info.DefaultSeries = defaultSeries
-		} else {
-			logger.Errorf("cannot get default series from base %q: %v", defaultBase, err)
-			// This is slightly defensive, but we should always show a series
-			// in the model info.
-			info.DefaultSeries = jujuversion.DefaultSupportedLTS()
-		}
```
Comment (SimonRichardson) [inline apiserver/facades/client/modelmanager/modelmanager.go:954]: :+1:

File: rpc/params/model.go [hunk 3]
```diff
@@ -145,8 +145,6 @@ type ModelInfo struct {
 	ControllerUUID     string `json:"controller-uuid"`
 	IsController       bool   `json:"is-controller"`
 	ProviderType       string `json:"provider-type,omitempty"`
-	DefaultSeries      string `json:"default-series,omitempty"` // default-series is deprecated, use default-base
-	DefaultBase        string `json:"default-base,omitempty"`
```
Comment (SimonRichardson) [inline rpc/params/model.go:149]: Are we sure that pylibjuju doesn't use this?

**Review by SimonRichardson — CHANGES_REQUESTED** [review]
Can we make sure that nobody cares about the default series/base on the model info.

**Review by SimonRichardson — APPROVED** [review]

**Pull request merged**